code/modules/clothing/gloves/_gloves.dm

```diff
@@ -11,6 +11,10 @@
 	var/transfer_prints = FALSE
 	strip_delay = 20
 	equip_delay_other = 40
+	/// Boolean. If TRUE this glove will produce a fingerless version when cut by wirecutters.
+	var/can_be_cut = FALSE
+	// if TRUE, will designate the result of cutting gloves
+	var/cut_type = FALSE
```

Comment:
```suggestion
	// if TRUE, will designate the result of cutting gloves
	var/cut_type = null
```
* We no longer need the boolean. If a glove can be cut, then it will have a `cut_type` path defined we can check for.
* `FALSE` is a boolean value, and `cut_type` stores paths. When it's empty, `null` is the correct type to represent the lack of it.